inspec/kubernetes/README.md

```diff
@@ -0,0 +1,12 @@
+To execute this container inside a kubernetes pod you have to set the variables for the profile of the environment you want to use
```

Comment:
Who needs this permission file? inspect as a service or you when executing kubectl commands or maybe something else? isn't this something that could be handled by a role/policy?